nix/packages.nix

```diff
@@ -150,11 +150,6 @@ in rec {
     pex = pkgs.callPackage ./tools/pex {};
     pipenv = pkgs.pipenv;
 
-    # Databases
-    cassandra = pkgs.cassandra;
```

Comment:
do also remove the wrappers in `dev-env/bin` that reference these.

Reply:
Good point, thanks! Done.